NightscoutServiceKit/NightscoutService.swift

```diff
@@ -57,13 +57,16 @@ public final class NightscoutService: Service {
         }
         return _uploader
     }
+    
```

Comment:
We will have 2 command sources eventually... One for Remote 1.0 and another for Remote 2.0. There will be different formats, validation mechanism and status reporting for each -- so it is nice to make dedicated classes for these types.